packages/react-dom-bindings/src/server/ReactFizzConfigDOM.js

```diff
@@ -101,6 +103,12 @@ export function prepareHostDispatcher() {
   ReactDOMCurrentDispatcher.current = ReactDOMServerDispatcher;
 }
 
+export type HeadersDescriptor = {
+  Link: string,
```

Comment:
Maybe this field should be optional just to indicate that in the future we might not include it in every call. Given that this type will be reflected in public types.

Reply:
good catch!